init/eessi_environment_variables

```diff
@@ -38,6 +38,14 @@ if [ -d $EESSI_PREFIX ]; then
           break
         fi
       done
+      export EESSI_ACCEL_PATH=$(${EESSI_INIT_DIR_PATH}/eessi_archdetect.sh accelpath)
+      if [ -z ${EESSI_ACCEL_PATH} ]; then
+          show_msg "archdetect could not find any accelerators"
+      else
+          if [ -d ${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR}/${EESSI_ACCEL_PATH} ]; then
```

Comment:
This assumes that their must be an accel arch for every CPU path, and doesn't allow for mix-and-match. Are we implementing things in such a strict manner?

Reply:
Well, right now the way this is implemented you can only completely override the CPU, not take one software stack for CPU and another (compatible) one for GPU. We could be more flexible:
```suggestion
          EESSI_GPU_SOFTWARE_SUBDIR=${$EESSI_GPU_SOFTWARE_SUBDIR_OVERRIDE:-${EESSI_SOFTWARE_SUBDIR}}
          EESSI_GPU_SOFTWARE_PATH=${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_GPU_SOFTWARE_SUBDIR}
          if [ -d ${EESSI_GPU_SOFTWARE_PATH}/${EESSI_ACCEL_PATH} ]; then
```